src/compiler/types.ts

```diff
@@ -2363,6 +2364,7 @@ namespace ts {
 
     export interface JSDocTemplateTag extends JSDocTag {
         kind: SyntaxKind.JSDocTemplateTag;
+        constraint: TypeNode | undefined;

```

Comment:
Only the first type parameter in a list is constrained by the constraint. Can you add a test for that? That is, `/** @template {number} T,U */` only constrains T.

Reply:
That was done in #24600 for the reason that reusing a node would be bad -- maybe not necessary now?

Reply:
The test `jsdocTemplateTag3`, added in #24600, will have fewer errors if this is changed, so it's already being tested.

Reply:
I don't like applying the constraint to multiple parameters even if there's no implementation reason that it's easier.